packages/pg-v5/test/commands/repoint.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+/* global describe it beforeEach afterEach */
+
+const cli = require('heroku-cli-util')
+const expect = require('unexpected')
+const nock = require('nock')
+const proxyquire = require('proxyquire')
+
+let addon
+const fetcher = () => {
+  return {
+    addon: () => addon
+  }
+}
+
+const cmd = proxyquire('../../commands/repoint', {
+  '../lib/fetcher': fetcher
+})
+
+describe('pg:repoint', () => {
+  let api, pg
+
+  beforeEach(() => {
+    addon = {
+      id: 1,
+      name: 'postgres-1',
+      plan: { name: 'heroku-postgresql:standard-0' }
+    }
+
+    api = nock('https://api.heroku.com')
+    pg = nock('https://postgres-api.heroku.com')
+    cli.mockConsole()
+  })
+
+  afterEach(() => {
+    nock.cleanAll()
+    api.done()
+    pg.done()
+  })
+
+  it('refuses to repoint hobby dbs', () => {
+    addon.plan = { name: 'heroku-postgresql:hobby-dev' }
+
+    return expect(cmd.run({ app: 'myapp', args: {}, flags: { confirm: 'myapp' } }),
+      'to be rejected with',
+      new Error('pg:repoint is only available for follower production databases'))
+  })
+
+  it('refuses to repoint non-follower dbs', () => {
+    pg.get('/client/v11/databases/1').reply(200, { forked_from: 'postgres://db1' })
+
+    return expect(cmd.run({ app: 'myapp', args: {}, flags: { confirm: 'myapp' } }),
+      'to be rejected with',
+      new Error('pg:repoint is only available for follower production databases'))
+  })
+
+  it('repoints db', () => {
+    api.get('/apps/myapp/config-vars').reply(200, { DATABASE_URL: 'postgres://db1' })
+    pg.get('/client/v11/databases/1').reply(200, { following: 'postgres://db1' })
+    pg.post('/client/v11/databases/1/repoint').reply(200)
+    return cmd.run({ app: 'myapp', args: {}, flags: { confirm: 'myapp', follow: 'NEW_LEADER' } })
+      .then(() => expect(cli.stderr, 'to equal', 'Starting repoint of postgres-1... heroku pg:wait to track status\n'))
```

Comment:
Optional: `run heroku pg:wait to track status` more clearer?

Reply:
I initially thought so too, but looking at upgrade we do
```js
cli.action.done(`${cli.color.cmd('heroku pg:wait')} to track status`)
```
so might as well stick with the same thing for now.

Reply:
yeah, this command is copied from `upgrade.js` - it's kinda the closest operation and I wanted to keep the same in style.